legacy_packages/sdk/src/evm/core/classes/internal/marketplace/marketplace-direct.ts

```diff
@@ -485,6 +503,9 @@ export class MarketplaceDirect {
       return Transaction.fromContractWrapper({
         contractWrapper: this.contractWrapper,
         method: "cancelDirectListing",
+        overrides: {
+          extraGas: BigInt(50000),
```

Comment:
nit: use BigNumber.from(50000) here since that's what we use everywhere in v4

also did you verify that fixes the issue in v4? reason im asking is because in v4 we do have already a gas bump for dynamic contracts IIRC

Reply:
or just `50000` since it later goes into `BigNumber.from()` anyways

Reply:
Haven't confirmed it yet, trying to now